test/cpp/jit/test_gpu.cpp

```diff
@@ -19604,6 +19604,31 @@ TEST(NVFuserTest, FusionNonDivisibleSplitVectorize2_CUDA) {
   testValidate(&fusion, cg_outputs, {t0}, {ref}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, Issue1305Repro_CUDA) {
```

Comment:
nit: `TEST_F(NVFuserTest, FusionIssue1305Repro_CUDA) {`.

I wonder we really need the `Fusion` suffix.

Reply:
Probably not but best to be consistent